bom/pom.xml

```diff
@@ -111,6 +113,7 @@
         <scalatest.version>3.2.17</scalatest.version>
         <docker-java.version>3.3.3</docker-java.version>
         <system-rules.version>1.19.0</system-rules.version>
+        <awssdk.version>2.26.21</awssdk.version>
```

Comment:
Could you please move this declared version above the "test" dependency versions, maybe directly below the mongodb version - and adding a comment that this is needed for MongoDB AWS IAM authentication.